doc/Type/Cool.pod6

```diff
@@ -1163,12 +1163,18 @@ L<Regex|/type/Regex> as a list of strings.
 
     multi method contains(Cool:D: Str(Cool) $needle, Cool $start? --> Bool:D)
 
+Coerces the first two arguments (in method form, also counting the invocant)
+to L<Str|/type/Str>, and searches for C<$needle> in the string starting from C<$start>.
 Returns C<True> if the invocant contains the C<$needle> at any position within
 the string. If $start is provided skip as many characters.
 
     say "Hello, World".contains('hello');      # OUTPUT: «False␤»
     say "Hello, World".contains(',');          # OUTPUT: «True␤»
 
+Note that because of how a L<List|/type/List> or L<Array|/type/Array> is 
+L<coerced|/type/List#method_Str> into a L<Str|/type/Str>, the results may 
+sometimes be surprising. See L<traps|/language/traps>.
```

Comment:
+1 : It would be nice to post an anchored link, i.e. /language/traps#coercion-with-lists or whatever the anchor on that page might be.